indra/ontology/bio/ontology.py

```diff
@@ -157,7 +163,8 @@ def add_famplex_nodes(self):
                 os.path.join('famplex', 'entities.csv')), delimiter=','):
             entity = row[0]
             nodes.append((self.label('FPLX', entity),
-                          {'name': entity}))
+                          {'name': entity,
+                           'type': 'protein_family_complex'}))
```

Comment:
I guess it's not so simple but it would be nice to get a mixture of strict protein families, strict complexes, and mixtures (i.e., famplexes) from the famplex resource

Reply:
That would be possible, I didn't do it here just because that further stratification wasn't needed for statistics but we could implement that. I was actually thinking of potentially adding another layer called subtype for things like this.

Reply:
> I guess it's not so simple but it would be nice to get a mixture of strict protein families, strict complexes, and mixtures (i.e., famplexes) from the famplex resource

I played around with implementing something like this when I made famplex into a python package last year. I just tried the simple thing where if all of the relationships beneath a term were `isa` then it's a family, if all of these edges are `partof`, then it's a complex, and if there's a mixture of edge types then it's a famplex. I'd written a function to find the type dynamically, but ended up not including it.